onnxruntime/test/providers/cpu/math/element_wise_ops_test.cc

```diff
@@ -970,6 +970,17 @@ TEST(MathOpTest, Expand_8_1x3_int64) {
   test.Run();
 }
 
+TEST(MathOpTest, Expand_8_3x1x3x1_int64) {
+  OpTester test("Expand", 8);
+  test.AddInput<int64_t>("data_0", {1, 3, 1, 3}, {1, 2, 3, 4, 5, 6, 7, 8, 9});
+  test.AddInput<int64_t>("data_1", {4}, {3, 1, 3, 1});
+  test.AddOutput<int64_t>("result", {3, 3, 3, 3},
+	  {1, 2, 3, 1, 2, 3, 1, 2, 3, 4, 5, 6, 4, 5, 6, 4, 5, 6, 7, 8, 9, 7, 8, 9, 7, 8, 9,
```

Comment:
>	 [](start = 0, length = 1)

nit: tabs

Reply:
fixed

---
In reply to: [297307894](https://github.com/microsoft/onnxruntime/pull/1292#discussion_r297307894) [](ancestors = 297307894)